### PR TITLE
Molving to preprocessify instead of remove lines since it's more vers…

### PR DESCRIPTION
**Pull request by arcturus**
…atile

Also checking if we have performance.mark in order to not to fail in chrome

### CODE REVIEW

**Comment by arcturus** [comment]
@lodr r?

File: gulpfile.js [hunk 1]
```diff
@@ -26,19 +26,15 @@ gulp.task('bundle-dist', function() {
   var bundler = browserify({
     entries: ['./index.js'],
     debug: false,
-    standAlone: 'ServiceWorkerWare'
+    standAlone: 'ServiceWorkerWare',
```
Comment (lodr) [inline gulpfile.js]: Beware of that colon at the end of the line.

File: gulpfile.js [hunk 2]
```diff
@@ -55,6 +51,9 @@ gulp.task('bundle-debug', function() {
 
   var bundle = function() {
     return bundler
+      .transform(preprocessify(
+        { DEBUG: true }
```
Comment (lodr) [inline gulpfile.js]: What about having `PERF` and `DEBUG` so in production we can have both disabled and remove performance marks as well as debug traces? If you like, open a new issue with the change.

**Comment by lodr** [comment]
I like this preprocessor. It's a little bit verbose to enclose all with the directives and it would be great to have [a special syntax for the directives to affect next line / block only](https://github.com/jsoverson/preprocess/issues/94).

File: gulpfile.js (hunk 2 above)
Comment (arcturus) [inline gulpfile.js]: Agree, I was thinking on it, creating a debug one for the debug messages and the one for perf.

We can add it in a different patch, specially if your proposition on the preprocessing repo is accepted, will be pretty easy.

File: gulpfile.js (hunk 1 above)
Comment (arcturus) [inline gulpfile.js]: right!

**Pull request merged**